packages/babel-preset/index.js

```diff
@@ -73,22 +77,12 @@ function getTargets(
 }
 
 function preset(api, explicitOptions = {}) {
-  // FIXME: This is meant to default to production, but Babel actually defaults
-  //  us to development.
-  const env = api.env(); // || 'production';
```

Comment:
we could just use `getEnv('production')` from `@babel/core`. the only downside is that if someone does `babel --env-name development`, it won't work... but maybe that's okay?

Reply:
I think it's better to decouple these entirely. We use envName already for conditional esm compilation in libraries and that conflicts here. You may want dev and esm build. react-preset already does it like this so it seems like a good idea

Reply:
alternatively create-react-app does an env thing but insists it's set: https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/create.js

Reply:
all i want is to ensure that `dev` is not the default, and honestly this may only be relevant for `web-app` targets right?

Reply:
well, you do have the handling for `node`

Reply:
right, i guess it's really the `app` target of which we have no node one